examples/llm/inference/convert_hf_to_onnx.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Basic HuggingFace -> ONNX export script.
+
```

Comment:
Add some example of invoking this script here like the ones we had in the previous script.

Reply:
will do